.github/workflows/test-core.yaml

```diff
@@ -432,8 +402,6 @@ jobs:
     runs-on: windows-2022
     env:
       SOTER_KDF_RUN_LONG_TESTS: yes
-      # TODO: stop using deprecated API so that warnings can be errors again
-      WITH_EXPERIMENTAL_OPENSSL_3_SUPPORT: yes
       WITH_FATAL_WARNINGS: "no" # YAML :trollface:
```

Comment:
Missed this line, will remove in next commit. BTW, why [MSYS2 environment](https://github.com/cossacklabs/themis/actions/runs/4326599795/jobs/7554211265) job fails like there is still "no support for OpenSSL 3" in header file?

Reply:
> why [MSYS2 environment](https://github.com/cossacklabs/themis/actions/runs/4326599795/jobs/7554211265) job fails like there is still "no support for OpenSSL 3" in header file?

Hm... Looks like the package uses the pristine 0.14.0 tarball, not the source from the current tree.

That is, when you build a package, it's built locally but still references the tarball from release artifacts, not packages the source from whatever the current branch contains. And it's been doing that for a while. Says a lot about how well-maintained MSYS2 builds are.

Uh, well... I think you should leave `WITH_EXPERIMENTAL_OPENSSL_3_SUPPORT: yes` here for the time being. Then *someone* 🙄 should apply whatever hacks necessary for this job to package the tree being tested.

Reply:
Kinda weird seeing Pacman being used on Windows. Anyway, added this line back and another comment instead, since "stop using deprecated API" is not relevant anymore